packages/behandling-pleiepenger/src/panelDefinisjoner/prosessStegPaneler/VedtakProsessStegPanelDef.tsx

```diff
@@ -25,15 +25,21 @@ class PanelDef extends ProsessStegPanelDef {
     aksjonspunktCodes.KONTROLL_AV_MAUNELT_OPPRETTET_REVURDERINGSBEHANDLING,
   ];
 
-  getEndepunkter = () => [
-    PleiepengerBehandlingApiKeys.TILBAKEKREVINGVALG,
-    PleiepengerBehandlingApiKeys.SEND_VARSEL_OM_REVURDERING,
-    PleiepengerBehandlingApiKeys.MEDLEMSKAP,
-    PleiepengerBehandlingApiKeys.VEDTAK_VARSEL,
-    PleiepengerBehandlingApiKeys.TILGJENGELIGE_VEDTAKSBREV,
-    PleiepengerBehandlingApiKeys.INFORMASJONSBEHOV_VEDTAKSBREV,
-    PleiepengerBehandlingApiKeys.DOKUMENTDATA_HENTE,
-  ];
+  getEndepunkter = (featureToggles: FeatureToggles) =>
+    [
+      PleiepengerBehandlingApiKeys.TILBAKEKREVINGVALG,
+      PleiepengerBehandlingApiKeys.SEND_VARSEL_OM_REVURDERING,
+      PleiepengerBehandlingApiKeys.MEDLEMSKAP,
+      PleiepengerBehandlingApiKeys.VEDTAK_VARSEL,
+    ].concat(
+      featureToggles?.DOKUMENTDATA
```

Comment:
Dette er dessverre en utgått featuretoggle, som burde ha vært slettet. Den var bare i bruk for frisinn